index.src.html

```diff
@@ -3624,6 +3647,12 @@
 
     When invoked, run the [=Close VideoFrame=] algorithm with [=this=].
 
+: <dfn method for=VideoFrame>metadata()</dfn>
+:: Gets the {{VideoFrameMetadata}} associated with this frame.
```

Comment:
Is there a reason you went with this approach, versus the exception that all the other methods throw when closed? I think it's more error prone to return an empty/fresh structure versus throwing an exception.

I think you should instead throw an exception and assign null during close() algorithm unless there's a good reason otherwise.

Reply:
Right, now that we moved from an attribute to a method, it makes sense to throw for consistency.

Reply:
Done